src/Oliver.Client/Executing/Executor.cs

```diff
@@ -59,12 +63,21 @@ private async Task Execute(Configurations.Client.Instance instance, Execution ex
                 var logs = new List<string> { $"Starting step {step.Order}: '{step.Name}'" };
 
                 var command = Substitute(step.Command, variables?.Values);
-                var folder = Substitute(step.WorkingFolder, variables?.Values);
+
+                var folder = Substitute(step.WorkingFolder ?? "", variables?.Values);
```

Comment:
`WorkingFolder` already has the default value so there is no need to add it again.

Reply:
done